js/calculator.js

```diff
@@ -0,0 +1,38 @@
+const add = (a, b) => {
+  let message = null;
+  if (typeof a === 'string' || typeof b === 'string') {
+    message = 'Input has to be an integer';
+  } if (typeof a === 'number' && typeof b === 'number') {
+    message = a + b;
+  }
+  return message;
+};
+
+const multiplication = (a, b) => {
+  let message = null;
+  if (typeof a === 'number' && typeof b === 'number') {
+    message = a * b;
+  }
+  return message;
+};
+
+const subtract = (a, b) => {
+  let message = null;
+  if (typeof a === 'number' && typeof b === 'number') {
+    message = a - b;
+  }
+  return message;
+};
+
+const divide = (a, b) => {
+  let message = null;
+  if (typeof a === 'number' && typeof b === 'number') {
+    message = a / b;
+  }
+  return message;
+};
+
+const calc = {
+  add, multiplication, subtract, divide,
+};
+export default calc;
```

Comment:
#### `calculator` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/calculator_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/testing-practice))

**_Although you have all these functions available it should be within an object_**

* [ ] It should be an object
* [ ] It should have a `add` method
* [ ] It should have a `subtract`
* [ ] It should have a `multiply` method
* [ ] It should have a `divide` method